packages/theme/src/cli/utilities/theme-fs.ts

```diff
@@ -194,13 +194,20 @@ export function mountThemeFileSystem(root: string, options?: ThemeFileSystemOpti
     files.delete(fileKey)
     emitEvent('unlink', {fileKey})
 
-    deleteThemeAsset(Number(themeId), fileKey, adminSession)
-      .then(async (success) => {
-        if (!success) throw new Error('Unknown issue.')
-        outputSyncResult('delete', fileKey)
+    fetchThemeAsset(Number(themeId), fileKey, adminSession)
```

Comment:
I'm debating whether we should make this call explicitly, or just call `deleteThemeAsset` (which will fail) and handle the failures silently. 

This would save an additional call every time a file is deleted, but the tradeoff with that approach would mean that we are unable to discern between legitimate failures of `deleteThemeAsset` (such as a server failure) from instances where the failure comes from the asset not existing on the remote theme.